src/select.coffee

```diff
@@ -492,8 +531,10 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
 
   actions:
     toggleDropdown: (event) ->
-      return if @get('disabled')
-      @toggleProperty 'showDropdown'
+      if @get 'showDropdown'
+        @send 'hideDropdown' if not @get('lockDropdownOnShow')
```

Comment:
What do you mean? 
This is for the case of toggling the dropdown. Where we have an `else` case to open the dropdown if it's not there yet. Other logics are aiming to hide the dropdown only.
Is it the answer of your question?